Makefile

```diff
@@ -62,14 +64,16 @@ localintegration: all
 	bats -t tests/integration${TESTFLAGS}
 
 install:
-	install -D -m0755 runc /usr/local/sbin/runc
+	install -D -m0755 runc $(BINDIR)/runc
+	install -D -m0644 completions/bash/runc $(PREFIX)/share/bash-completion/completions/runc
```

Comment:
Not all users have bash (I don't, for example). This should either be a separate install target (like `install-man`, we could have `install-bash`) or be a separate makefile in `contrib/completions/bash`.